akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -353,6 +353,13 @@ private[remote] class Association(
               dropped(ControlQueueIndex, controlQueueSize, outboundEnvelope)
           case _ ⇒
             val queueIndex = selectQueue(recipient)
+
+            //            def msgClass = message match {
+            //              case ActorSelectionMessage(m, _, _) ⇒ m
+            //              case m                              ⇒ m
+            //            }
+            //            println(s"# selectQueue $msgClass -> $queueIndex") // FIXME
```

Comment:
I was wrong about the PriorityMessage, the HeartbeatRsp is also sent over the control stream. It's selected in the `selectQueue` method.

Reply:
but the new "crossDcHeartbeatSender" should also be added to prio wildcard index